packages/Webkul/Admin/src/Resources/lang/en/app.php

```diff
@@ -3159,6 +3159,7 @@
                 'new'                           => 'New',
                 'no'                            => 'No',
                 'parent-id'                     => 'Parent ID',
+                'parent-id-hint'                => 'You can enter multiple parent IDs as comma-separated values (e.g., 12,15,34)',
```

Comment:
Please provide translation for all locale.

Reply:
[Added translation entry for parent-id-hint in all locale files](https://github.com/bagisto/bagisto/pull/10614/commits/de0528fec8550353415e3ef0a22114a7ecc3a0a2)